src/pages/settings/common/hooks/useHandleCompanySave.tsx

```diff
@@ -34,29 +37,21 @@ export function useHandleCompanySave() {
       .then((response) => {
         dispatch(updateRecord({ object: 'company', data: response.data.data }));
 
-        toast.success(t('updated_settings'), { id: toastId });
+        queryClient.invalidateQueries('/api/v1/statics');
+
+        Statics.reloadQuery();
+
+        toast.success('updated_settings');
       })
       .catch((error: AxiosError<ValidationBag>) => {
-        console.error(error);
-
-        toast.error(t('error_title'), { id: toastId });
+        if (error.response?.status === 422) {
+          toast.error(error.response.data.message);
```

Comment:
As we discussed earlier on the Slack, this way of handling 422 errors is not good. So, I changed the package we used for toasting and removed this handling. In my opinion, this part of the refactoring is not related to this PR. We use this `useHandleCompanySave` hook on like 10 different components, so the refactoring will involve many more files. I recommend creating an issue on Jira for this refactoring and then we can end up with 422 errors for all components. But, let me know your thoughts.

Reply:
>  In my opinion, this part of the refactoring is not related to this PR.

Exactly. Please move this to another pull request.